MainView.ux

```diff
@@ -3,14 +3,27 @@
 		<Material.Drawer ux:Binding="Drawer" Edge="Left">
 			<Text Alignment="Center">Hello!</Text>
 		</Material.Drawer>
-			
-		<DockPanel>
 
+		<DockPanel>
 			<Material.AppBar Dock="Top">
 				<Material.AppBarIcon Dock="Left" Icon="Menu" Clicked="{openDrawerPanel}" />
 				<Material.AppBarIcon Dock="Right" Icon="DotsVertical" Clicked="{toggleMenu}" />
 				<Material.AppBarIcon Dock="Right" Icon="AccountPlus"  />
 			</Material.AppBar>
+
+			<Rectangle Dock="Top" Margin="5">
+				<StackLayout/>
```

Comment:
I just wanted to use a stroke to separate the button example area from other upcoming examples